package-lock.json

```diff
@@ -721,9 +721,9 @@
       }
     },
     "auth0-source-control-extension-tools": {
-      "version": "4.0.4",
-      "resolved": "https://registry.npmjs.org/auth0-source-control-extension-tools/-/auth0-source-control-extension-tools-4.0.4.tgz",
-      "integrity": "sha512-k27PuMyzDthQeXIX+uEvNXpIU+1DnPjga08yGj4cyhEXmFyA1x6wcAdK2YeO5jBwzSverTM90vU/zQZNDT5lDQ==",
+      "version": "4.0.5",
+      "resolved": "https://a0us.jfrog.io/a0us/api/npm/npm/auth0-source-control-extension-tools/-/auth0-source-control-extension-tools-4.0.5.tgz",
```

Comment:
A few artifactory stuff got through here, should update or remote your .npmrc file.

Reply:
Done.